server/src/Metrics.hs

```diff
@@ -122,5 +125,8 @@ incrementWsQueueAdded = incCounter . icepeakMetricsWsQueueAdded
 incrementWsQueueRemoved :: MonadMonitor m => IcepeakMetrics -> m ()
 incrementWsQueueRemoved = incCounter . icepeakMetricsWsQueueRemoved
 
+incrementWsQueueSkippedUpdates :: MonadMonitor m => IcepeakMetrics -> m ()
+incrementWsQueueSkippedUpdates = incCounter . icepeakMetricsWsQueueSkippedUpdates
```

Comment:
This function appears unused - I think this is the one you meant to use for the other queue now, given you resolved my other comment about this?